wire-ios-data-model/Source/E2EIdentity/E2eISetupService.swift

```diff
@@ -22,7 +22,14 @@ import WireCoreCrypto
 public protocol E2eISetupServiceInterface {
 
     func registerTrustAnchor(_ trustAnchor: String) async throws
-    func setupEnrollment(userName: String, handle: String, teamId: UUID) async throws -> E2eiEnrollment
+
+    func setupEnrollment(
```

Comment:
It would be nice to have a documentation at least about the argument `isUpgradingClient`. 
Something like

```suggestion
    /// Sets up ...
    /// - parameter isUpgradingClient: `true` has the effect of ...
    func setupEnrollment(
```

Reply:
Done